framework/packages/abstract-adapter/Cargo.toml

```diff
@@ -11,6 +11,13 @@ resolver = "2"
 crate-type = ["cdylib", "rlib"]
 
 [features]
+default = ["ibc"]
```

Comment:
if we feature flag it in `abstract-adapter`, we need to feature flag it in `abstract-sdk` and `abstract-std`, to disable message variants